packages/shared/src/buttons/IncrementDecrementButtons.js

```diff
@@ -45,78 +35,60 @@ const Button = ({
 };
 
 export default function IncrementDecrementButtons(props: {|
-  onIncrement: () => void,
-  onDecrement: () => void,
-  number: number,
-  min?: number,
-  max?: number,
+  +onIncrement: () => void,
+  +onDecrement: () => void,
```

Comment:
Idem here.

Reply:
What?

Reply:
Oh it looks like my other comment was not taken into account: I meant for you to maybe extract the Props type to make it more readable.